packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -2062,7 +2062,10 @@ class IrisGrid extends Component<IrisGridProps, IrisGridState> {
               return;
             }
             const row = data.rows[0];
-            const values = keyTable.columns.map(column => row.get(column));
+            // Core JSAPI returns undefined for null table values, IrisGridPartitionSelector expects null
```

Comment:
Add the ticket number
```suggestion
            // Core JSAPI returns undefined for null table values, IrisGridPartitionSelector expects null (https://github.com/deephaven/deephaven-core/issues/5400)
```